test/src/mocha-utils.ts

```diff
@@ -224,6 +225,7 @@ export const mochaHooks = {
       state.puppeteerPath = path.resolve(
         path.join(__dirname, '..', '..', 'packages', 'puppeteer')
       );
+      state.protocol = protocol;
```

Comment:
Perhaps, let's not expose the protocol here as we don't want to grow the number of protocol/browser specific expectations in the tests. Perhaps, let's make sure that the errors are properly wrapped on the Puppeteer-level so that the tests can have a single assertion? cc @Lightning00Blade

Reply:
That of course would be even better!

Reply:
Should I create maybe an issue for it? 🤔